musicFunction.js

```diff
@@ -0,0 +1,52 @@
+var original;
+var i = 0;
+
+window.onload = function() {
+original = document.getElementById('container');
+
+};
+
+function playSound(id){
+	//duplicate 
+	var clone = original.cloneNode(true); // clone
+	clone.id = "container" + ++i; // there can only be one element with an ID
+	original.parentNode.appendChild(clone);
+	//play sound
+	sound(id);
+	
+}
+
+function sound(x){
```

Comment:
This is great!  Since the elseif's are only checking one value, would it be possible to use a Switch statement here?  Many find them a bit shorter and easier to follow, so it might be a good way to refactor!

Reply:
Whoops, looks like mxrosenthal beat me to it!

Reply:
Haha yes I think you are both correct :)